src/main/java/spoon/reflect/reference/CtWildcardReference.java

```diff
@@ -16,6 +16,9 @@
  */
 package spoon.reflect.reference;
 
+/**
+ * This interface defines a reference to a wildcard (<code>?</code>).
```

Comment:
Represents  a wildcard in generic type annotations, i.e. the "?" (e.g. the "?" in <code>Collection<?></code> or <code>Collection<? extends List></code>).